### PR TITLE
Add a metric to report crush weights for each OSDs

### PR DESCRIPTION
**Pull request by toshipp**
issue: https://github.com/cybozu-private/csa-apps/issues/2075

### CODE REVIEW

File: ceph-extra-exporter/executer_test.go [hunk 1]
```diff
@@ -9,68 +9,83 @@ import (
...
-				command: []string{"echo", autoscale_status_json},
+				name:    "test",
+				command: []string{"echo", `[{"key": "key1", "value": 1}, {"key": "key2", "value": 2}]`},
 				metrics: map[string]metric{
-					"pool_count": {
+					"sum": {
 						metricType: prometheus.GaugeValue,
-						help:       "pool count of `ceph osd pool autoscale-status` command",
-						jqFilter:   "[{value: . | length, labels: []}]",
+						help:       "bar",
```
Comment (peng225) [inline ceph-extra-exporter/executer_test.go:23]:
```suggestion
						help:       "foo",
```

File: ceph-extra-exporter/executer_test.go [hunk 2]
```diff
@@ -9,68 +9,83 @@ import (
...
+						jqFilter:   "[{value: . | map(.value) | add, labels: []}]",
 					},
-					"actual_capacity_ratio": {
+					"value": {
 						metricType: prometheus.GaugeValue,
-						help:       "",
-						jqFilter:   "[.[] | {value: .actual_capacity_ratio, labels: [.pool_name]}]",
-						labelKeys:  []string{"pool_name"},
+						help:       "baz",
```
Comment (peng225) [inline ceph-extra-exporter/executer_test.go:28]:
```suggestion
						help:       "bar",
```

File: ceph-extra-exporter/executer_test.go [hunk 3]
```diff
@@ -9,68 +9,83 @@ import (
...
-				command: []string{"echo", autoscale_status_json},
+				name:    "test",
+				command: []string{"echo", `[{"key": "key1", "value": 1}, {"key": "key2", "value": 2}]`},
 				metrics: map[string]metric{
-					"pool_count": {
+					"sum": {
 						metricType: prometheus.GaugeValue,
-						help:       "pool count of `ceph osd pool autoscale-status` command",
-						jqFilter:   "[{value: . | length, labels: []}]",
+						help:       "bar",
```
Comment (peng225) [inline ceph-extra-exporter/executer_test.go:60]:
```suggestion
						help:       "foo",
```

**Review by ushitora-anqou — DISMISSED** [review]

**Review by peng225 — APPROVED** [review]

File: ceph-extra-exporter/executer_test.go [hunk 4]
```diff
@@ -9,68 +9,83 @@ import (
...
-				command: []string{"echo", autoscale_status_json},
+				name:    "test",
+				command: []string{"echo", `[{"key": "key1", "value": 1}, {"key": "key2", "value": 2}]`},
 				metrics: map[string]metric{
-					"pool_count": {
+					"sum": {
 						metricType: prometheus.GaugeValue,
-						help:       "pool count of `ceph osd pool autoscale-status` command",
-						jqFilter:   "[{value: . | length, labels: []}]",
+						help:       "foo",
```
Comment (toshipp) [inline ceph-extra-exporter/executer_test.go:60]: We do not specify help text in other places, should we remove this or specify texts in other places?

File: ceph-extra-exporter/main.go [hunk 5]
```diff
@@ -67,15 +79,12 @@ func init() {
...
 
-func main() {
-	port := flag.Uint("port", 8080, "port number")
-	doesRunRGWAdmin := flag.Bool("export-rgw-metrics", true, "to export RGW related metrics or not")
-	flag.Parse()
-
+func startServer(rules []rule, port uint, doesRunRGWAdmin bool) error {
 	wg := &sync.WaitGroup{}
 	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
```
Comment (toshipp) [inline ceph-extra-exporter/main.go:85]:
```suggestion
	defer func() {
		cancel()
		wg.Wait()
	}()
```

**Review by peng225 — APPROVED** [review]

**Pull request merged**